hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieMultiTableDeltaStreamer.java

```diff
@@ -135,6 +135,7 @@ private void populateTableExecutionContextList(TypedProperties properties, Strin
       if (cfg.enableMetaSync && StringUtils.isNullOrEmpty(tableProperties.getString(HoodieSyncConfig.META_SYNC_TABLE_NAME.key(), ""))) {
         throw new HoodieException("Meta sync table field not provided!");
       }
+      populateTransformerProps(cfg, tableProperties);
```

Comment:
Could you add a test for applying different transformers to different tables?

Reply:
@yihua Sure. I'm new to this. I'll work over this weekend.